packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1505,6 +1514,22 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.services.TextInputConfiguration.enableIMEPersonalizedLearning}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@template flutter.widgets.EditableText.spellCheckConfiguration}
+  /// Configuration that details how spell check performed.
+  ///
+  /// Specifies the [SpellCheckService] used to spell check text input, the
+  /// [SpellCheckSuggestionsHandler] used to style text with misspelled words,
+  /// and the [TextStyle] that handler uses to indicate misspelled words.
+  /// {@endtemplate}
+  ///
+  /// If the [SpellCheckService] is left null, the [DefaultSpellCheckService] if
+  /// used if the platform is supported. It is currently supported only on
+  /// Android. If the [SpellCheckSuggestionsHandler] is left null, the
+  /// [DefaultSpellCheckSuggestionsHandler] is used. If neither the
+  /// [SpellCheckService] nor the [SpellCheckSuggestionsHandler] is specified,
+  /// then spell check is disabled by default.
```

Comment:
Also if spellCheckConfiguration is null itself then spell check is disabled, right?  Maybe consider mentioning that here.